home/views/admin.py

```diff
@@ -12,7 +12,7 @@
 from crispy_forms.utils import render_crispy_form
 from discord_webhook import DiscordWebhook, DiscordEmbed
 
-from home.models import Review, Professor, ProfessorAlias, ProfessorCourse, Grade, User
+from home.models import Review, Professor, ProfessorCourse, ProfessorAlias, Grade, User
```

Comment:
this is switching import names for no reason as far as I can see? Can you revert this? it's just cluttering the diff